src/registrar/templates/domain_security_email.html

```diff
@@ -21,7 +21,7 @@ <h1>Domain security email</h1>
     <button
       type="submit"
       class="usa-button"
-    >Add security email</button>
+    >{% if form.security_email.value is None or form.security_email.value == "dotgov@cisa.dhs.gov"%}Add security email{% else %}Save{% endif %}</button>
```

Comment:
Did you see this in the figma from design somewhere? Or was this an ask from the design team? This feels off for our flow and normally I would just expect it to be a save button and add if we are generating a new field.

Reply:
Yup... An ask from the design team:
https://cisa-corp.slack.com/archives/C05BDEA3C11/p1695069485950669